test/cmdClientTest.js

```diff
@@ -161,6 +161,9 @@ describe('Running bin cmd client', function() {
     it('Send messages sasl', function(done) {
         verify(done, [example('../bin/sender-client.js', ['--broker', 'amqp://admin:admin@127.0.0.1:5672', '--count', 10, '--msg-content', 'msg no.%d'])], 0);
     });
+    it('Send messages sasl (username as email)', function(done) {
+        verify(done, [example('../bin/sender-client.js', ['--broker', 'amqp://admin@pepa.cz:admin@127.0.0.1:5672', '--count', 10, '--msg-content', 'msg no.%d'])], 0);
```

Comment:
@kornys I just saw you already created the test now. Just so you know, I am encoding the username on my test, it would be: "%40" instead of "@" character. In this case: "admin%40pepa.cz".

Reply:
@fgiorgetti  many thanks! now it is ok. Im going to merge it and publish in npm.